src/Data/LCA/Online.hs

```diff
@@ -56,12 +57,13 @@ import Data.LCA.View
 
 import Prelude hiding
   ( drop
-#if __GLASGOW_HASKELL__ < 710
   , length
   , null
-#endif
   )
 
+-- $setup
+-- >>> let length = Data.LCA.Online.length
```

Comment:
Why does this need to be here if `length` is hidden in the imports?

Reply:
`cabal-docspec` doesn't see imports. It `:m +Data.LCA.Online` and then the example uses `length` which then conflicts with `Prelude.length`.

Reply:
Ah, OK—I wasn't aware of that nuance of `cabal-docspec`.